var/spack/repos/builtin/packages/darshan-util/package.py

```diff
@@ -71,3 +71,40 @@ def configure_args(self):
                 extra_args.append('--enable-apxc-mod')
 
         return extra_args
+
+    @run_after('install')
+    def _copy_test_inputs(self):
+        # add darshan-test/example-output/mpi-io-test-spack-expected.txt"
+        test_inputs = \
+            ["darshan-test/example-output/mpi-io-test-x86_64-{0}.darshan"
```

Comment:
You could add a method -- with the `property` directive -- that builds this relative path and use it here and in `_test_parser`. 

We also encourage use of `join_path()` over using forward slashes for building paths since we have people working on Windows ports.

Reply:
Updated with a new commit to address those comments.